src/App.scss

```diff
@@ -73,84 +90,140 @@ body {
     margin-left: 50px;
 }
 
-.App__program-section {
-    margin-top: 1.5rem;
-}
-
 .App__command-palette {
-    background-color: #F1F1F1;
-    height: 100%;
-    padding-top: 2rem;
+    background-color: #30444E;
+    border-radius: 0.2rem;
     text-align: center;
+    margin: 0 0.5rem;
+    grid-column-start: 2;
+    grid-column-end: 3;
+    grid-row-start: 2;
+    grid-row-end: 4;
+    overflow: hidden;
 }
 
 .App__command-palette-heading {
-    font-weight: bold;
+    color: #F1AE5B;
+    font-weight: normal;
     font-size: inherit;
-    margin: 0 0 2rem 0;
+    margin: 1.5rem 0;
+    border-radius: 3px;
+}
+
+.App__commandpalette-heading-short {
+    display: none;
+}
+
+.App__command-palette-command-container {
+    width: 100%;
+    height: 100%;
+    padding: 1rem;
+    overflow-y: scroll;
```

Comment:
I observed this weird behaviour for Light theme in chrome browser, where the movements panel's width changes from when light theme is initially selected and then hovering over to one of movements commands, or clicking to the theme selector or the audio toggle switch. It appears to be not calculating width of the movements panel with the scroll bar when light theme is initially selected, and then adjust width with the scroll bar after an interaction with the components in the panel. I tried to find ways to resolve this issue, but best one I found was to set webkit-scrollbar property on this container with value initial; however, this fix hides the scroll bar on light theme. 

```
.App__command-palette-command-container::-webkit-scrollbar {
    appearance: initial;
}
```

Reply:
We should discuss this in our next meeting, either we need a proper fix or we need to decided which side effect to live with.